lib/middleware/karma.js

```diff
@@ -232,5 +232,14 @@ var createKarmaMiddleware = function (
   }
 }
 
+createKarmaMiddleware.$inject = [
+  'filesPromise',
+  'serveStaticFile',
+  'serveFile',
+  'injector',
+  'config.basePath',
+  'config.urlRoot'
+]
+
```

Comment:
@dignifiedquire wallaby can't parse special comment like   /* config.basePath */ in arguments of the function